src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -490,7 +487,7 @@ slots:
     range: string
     multivalued: true
   other:
-    description: Pipe separated list of key value pairs for properties not part of
+    description: List of key-value pairs for properties not part of
```

Comment:
1. If that slot is intended to be a list, then it should be `multivalued: true`.
2. Somewhere in the spec you need to specify how those key-value pairs are represented. I suppose they should be of the form `key=value`, but you don’t want to leave implementers guessing. (Unless Linkml already has a type for key-value pairs, in which case that type should be used here instead of `string`.)

Reply:
To clarify: the change I’m requesting for this PR is the first one (setting the slot to be multivalued). I don’t mind if the details of the serialisation of key-value pairs are left to be fixed later in another PR.

Reply:
Having key pairs formally represented in the model is indeed the best solution. I didn’t suggest it because I thought it would be too big of a change, I hadn’t noticed there was already a PR for that.

Reply:
However, this whole “extension” thing is going to be a nightmare for implementations in strongly typed languages…